ci/azure-install-rust.yml

```diff
@@ -4,7 +4,7 @@ steps:
       set -e
       curl https://sh.rustup.rs -sSf | sh -s -- -y --default-toolchain none
       export PATH=$PATH:$HOME/.cargo/bin
-      rustup toolchain install $RUSTUP_TOOLCHAIN
+      rustup toolchain install $RUSTUP_TOOLCHAIN --force
```

Comment:
Apparently there are already some components install on the Azure Pipelines agents, so in cases where toolstates are broken (like now) rustup is dropping backing to the previous nightly which results in the rustc-ap-* crate compile failures

https://dev.azure.com/rust-lang/rls/_build/results?buildId=36426&view=logs&j=846ec57a-2bea-5833-7752-007160b95b9a&t=58951315-86de-5a6f-15b3-9dd096725307&l=54
https://dev.azure.com/rust-lang/rls/_build/results?buildId=36426&view=logs&j=846ec57a-2bea-5833-7752-007160b95b9a&t=58951315-86de-5a6f-15b3-9dd096725307&l=33